modules/ztp-preparing-the-ztp-git-repository.adoc

```diff
@@ -62,35 +61,39 @@ For example:
 [source,text]
 ----
 example/
-  ├── policygentemplates
+  ├── acmpolicygenerator
+  │   ├── kustomization.yaml
+  │   └── source-crs/
+  ├── policygentemplates <1>
   │   ├── kustomization.yaml
   │   └── source-crs/
   └── siteconfig
         ├── extra-manifests
         └── kustomization.yaml
 ----
+<1> Using `PolicyGenTemplate` CRs to manage and deploy polices to managed clusters is deprecated in {product-title} 4.16. Equivalent and improved functionality is available using {rh-rhacm-first} and `PolicyGenerator` CRs.
```

Comment:
PolicyGenTemplate is not deprecated in 4.16. I would add a statement indicating that PolicyGenTemplate will be deprecated in the future